### PR TITLE
Adding new constructor

### PR DESCRIPTION
**Pull request by AntoninKns**
Adding new constructor requiring only the name of the problem, the group is then found automatically.

### CODE REVIEW

**Review by dpo — COMMENTED** [review]
Why not remove the old constructor? It doesn't have any advantage, does it?

**Comment by AntoninKns** [comment]
> Why not remove the old constructor? It doesn't have any advantage, does it?

I kept it only to avoid breaking other existing packages. That being said I think I'm the only one to use it yet so I can remove it.

**Comment by dpo** [comment]
you can deprecate it just to be careful: https://docs.julialang.org/en/v1/base/base/#Base.@deprecate

File: src/BundleAdjustmentArtifactFunctions.jl [hunk 1]
```diff
@@ -94,6 +135,26 @@ function fetch_ba_group(group::AbstractString)
   return problem_paths
 end
 
+"""
+    BundleAdjustmentModel(name::AbstractString; T::Type=Float64)
+
+Alternate constructor of BundleAdjustmentModel.
+Get the path of the problem name `name` with the precision `T`. 
+It will automatically find the group it belongs to.
+Return a NLSModel generated from this problem data using NLPModels
```
Comment (dpo) [inline src/BundleAdjustmentArtifactFunctions.jl:144]:
```suggestion
Return a NLSModel for problem `name` with precision `T`.
```

**Review by dpo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**